Folder convert branch

dan-c-underwood opened the pull request:
Merging the changes in the build system and folder structure

luke5542 committed:
initial folder changes

luke5542 committed:
added in all scripts to follow new setup

luke5542 committed:
finished initial run-script setup

luke5542 committed:
added pretty colors to run/build scripts

luke5542 committed:
added new compile/run script to support future setups

luke5542 committed:
fully changed around the the project to use the new folder setup.

luke5542 committed:
improved readme and added sieveSize selection to the script

georgevanburgh committed:
Fixed Dockerfile - can now build any branch

luke5542 committed:
removed rust lock

luke5542 committed:
readded Cargo's lock file

luke5542 committed:
added contributer on rust

georgevanburgh committed:
Replaced bash-based build system with Makefiles.

georgevanburgh committed:
Updated Dockerfile in response to build system changes

georgevanburgh committed:
Add .dockerignore file

georgevanburgh committed:
Merge remote-tracking branch 'upstream/master'

georgevanburgh committed:
Merged folder_convert_branch into master

Commit:
Updated README.md to reflect build system changes

georgevanburgh committed:
Fixed Dockerfile typo

georgevanburgh committed:
Merge branch 'master' of https://github.com/FireEater64/LanguageComparison

Commit:
Missed a few legacy build/run scripts on last commit

Commit:
Added basic compile-time checks for various runtimes

Commit:
Cleaned up buildall and cleanall outputs

Commit:
Improved handling for missing runtimes.

Top level makefile will now only target projects which have all their EXECUTABLEs
installed - otherwise these projects will be silently skipped.

Commit:
Improved handling for missing runtimes.

Top level makefile will now only target projects which have all their EXECUTABLEs
installed - otherwise these projects will be silently skipped.

Commit:
Rust Makefile should search for `rustc`, not `rust`

dan-c-underwood committed:
Merge pull request #13 from FireEater64/master

Switched build systems

Commit:
Added basic golang implementation for the sieve of erathosthenes.

dan-c-underwood committed:
Merge pull request #15 from FireEater64/master

Added basic golang implementation for the sieve of erathosthenes.

luke5542 committed:
Merge branch 'folder_convert_branch'

luke5542 committed:
Merge branch 'master' into folder_convert_branch

luke5542 commented:
Before merging, this should be improved to have the python benchmarking scripts run correctly to create the html graph.

Commit:
Fixed Docker issues, and golang project structure

The previous Dockerfile did not correctly install golang, and the
resultant image had an issue with the use of ```$(shell)``` to invoke
getBuildableSolutions.sh

The golang project structure has be reformatted, per the Google
guidelines - and make file adjusted accordingly

Commit:
Missed files from previous commit

dan-c-underwood committed:
Fixed python data script to work with new folder structure

dan-c-underwood committed:
Fixing label on axis

luke5542 committed:
Merge pull request #18 from FireEater64/master

Merged in the fixed Docker issues, and golang project structure

luke5542 committed:
fixed README

luke5542 committed:
Merge branch 'folder_convert_branch' of https://github.com/luke5542/LanguageComparison into folder_convert_branch

dan-c-underwood committed:
Preemptive bug fixes

dan-c-underwood committed:
Merge branch 'folder_convert_branch' of https://github.com/luke5542/LanguageComparison into folder_convert_branch
